packages/charts/react-charts/library/src/components/GaugeChart/useGaugeChartStyles.styles.ts

```diff
@@ -31,8 +31,8 @@ const useStyles = makeStyles({
     display: 'flex',
     flexDirection: 'column',
     alignItems: 'center',
-    width: '100%',
-    height: '100%',
+    width: 'var(--root-width, 100%)',
+    height: 'var(--root-height, 100%)',
```

Comment:
what does this change do

Reply:
ResponsiveContainer dynamically adjusts the width and height of the root div in the child chart component to ensure it resizes properly. In v8, we pass these width and height values directly through the styles prop. But in v9, the styles prop expects classNames instead. Since Griffel doesn't support generating classNames with dynamic values, and adding a new prop just for styles would not be ideal, we decided to use CSS variables to update the width and height of the root div.